core/src/test/java/org/apache/calcite/rex/RexProgramBuilderBase.java

```diff
@@ -756,8 +759,8 @@ protected RexNode vParam(String name, RelDataType type) {
    */
   protected RexNode vParam(String name, int arg, RelDataType type) {
     assertArgValue(arg);
-    RelDataType nonNullableType = typeFactory.createTypeWithNullability(type, false);
-    return rexBuilder.makeFieldAccess(getDynamicParam(nonNullableType, name), arg);
+    RelDataType nullableType = typeFactory.createTypeWithNullability(type, true);
```

Comment:
This line code is necessary? Because we have changed the code logic, The Test still passes, So this is an invalid code？

Reply:
I checked `getDynamicParam()` in `vParam`, which creates nullableType internally. So we don't need to change these codes.